ios/ImagePickerManager.m

```diff
@@ -224,6 +224,9 @@ - (void)onVideoObtained:(NSURL *)url
     }
     NSDictionary *response = @{@"uri": videoDestinationURL.absoluteString};
 
+    AVAsset *asset = [AVAsset assetWithURL:videoDestinationURL];
+    self.response[@"duration"] = @(roundf(CMTimeGetSeconds(asset.duration)));
+    self.response[@"uri"] = videoDestinationURL.absoluteString;
```

Comment:
```
ios/ImagePickerManager.m:228:10: Property 'response' not found on object of type 'ImagePickerManager *'
ios/ImagePickerManager.m:229:10: Property 'response' not found on object of type 'ImagePickerManager *'
```

Reply:
right, need to update those after the merge. I'll update it sometime later this afternoon and ping you then. Sorry for the hassle @Johan-dutoit

Reply:
I made the change, just check if you're happy.

Reply:
amazing, thanks